src/compute/src/logging/compute.rs

```diff
@@ -130,6 +131,38 @@ pub enum ComputeEvent {
     },
     /// A dataflow export was hydrated.
     Hydration { export_id: GlobalId },
+    /// An LIR operator was mapped to some particular dataflow operator.
+    ///
+    /// Cf. `ComputeLog::LirMaping`
+    LirMapping {
+        /// The `GlobalId` in which the LIR operator is rendered.
+        ///
+        /// NB a single a dataflow may have many `GlobalId`s inside it.
+        /// A separate mapping (using `ComputeEvent::DataflowGlobal`)
+        /// tracks the many-to-one relationship between `GlobalId`s and
+        /// dataflows.
+        global_id: GlobalId,
+        /// The LIR identifier (local to `export_id`).
+        lir_id: LirId,
+        /// The LIR operator, as a string (see `FlatPlanNode::humanize`).
+        /// We use `Box<str>` to reduce the size of the `ComputeEvent` representation.
+        operator: Box<str>,
+        /// The LIR identifier of the parent (if any).
+        /// Since `LirId`s are strictly positive, Rust can steal the low bit.
+        /// TODO(mgree) write a test to ensure that low bit is stolen
+        parent_lir_id: Option<LirId>,
```

Comment:
I think it's something we can rely on:
```
struct LirId(std::num::NonZeroUsize);

fn main() {
    println!("{}", std::mem::size_of::<Option<LirId>>());
}
```
prints 8.

Reply:
Adding a test for this, just to be safe.